reconcile/utils/terrascript_client.py

```diff
@@ -160,7 +160,7 @@ def __init__(self, integration, integration_prefix,
                 access_key=config['aws_access_key_id'],
                 secret_key=config['aws_secret_access_key'],
                 version=self.versions.get(name),
-                region=config['resourcesDefaultRegion'])
+                region=config['region'])
```

Comment:
aren't you reverting the previous fix here ?

Reply:
good catch!